### PR TITLE
Modified secondary tags and added new requirements

### PR DESCRIPTION
**Pull request by EthanDobbs**
Secondary Tags were not centered with the edge of the resource, but that is now fixed. Card resources can now had the class "card-resource--has-secondary-tag" to give them a wider right-side margin.

### CODE REVIEW

File: src/common/cards/RequirementType.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ export enum RequirementType {
     TAG = 'tag',
     PRODUCTION = 'production',
     REMOVED_PLANTS = 'Removed plants',
+    SUM_TAGS = 'Sum tags',
```
Comment (kberg) [inline src/common/cards/RequirementType.ts:12]: My mistake, these shouldn't be part of the PR since they're not being used in the app

**Comment by EthanDobbs** [comment]
Removed new requirements

**Review by kberg — APPROVED** [review]
I made a couple of modifications, but this is

**Comment by kberg** [comment]
... but this is good.
